### DISCUSSION TITLE
Can anyone explain the new settings - model page please?

### DISCUSSION THREAD

**Discussion by andreascschmidt** [open-webui/open-webui#7128] [General]
I see my existing models, but nothing like the old UI where I can search for new / add / remove the models. 
I can only disable them. 

I'm sure this page is supposed to be used somehow, or not ?

**Comment by tjbck** [comment]
Related: https://github.com/open-webui/open-webui/discussions/7099

**Comment by andreascschmidt** [comment]
Sorry, found it. Thanks
